deploy/charts/kiali-operator/values.yaml

```diff
@@ -0,0 +1,59 @@
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  repo: ${$OPERATOR_QUAY_NAME}
+  tag: ${$OPERATOR_QUAY_TAG}
```

Comment:
I don't see how this could ever work - this looks wrong with `${$`. I think this should be `${OPERATOR...`